site/src/schemas/component.schema.json

```diff
@@ -52,7 +52,7 @@
           "openUIName": { "type": "string" },
           "image": { "type": "string" }
         },
-        "required": ["name", "image"]
```

Comment:
Having "image" as a required property on the concept object generates 576 warnings on the current dataset.

Reply:
I have mixed feelings about this change.  Many of the pages are less useful than they should be because we don't have images.  But I also recognize that it's hard to produce all the necessary images and we're not always going to get them...

Reply:
In hindsight, it would maybe be better to keep it for encouraging adding images to future research instead of "fixing all the warnings". My initial thought was that it reflects the current state better.